src/artifacts/0.6/Artifacts.sol

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import { EntryPoint } from "@aa/0.6/contracts/core/EntryPoint.sol";
+import { SimpleAccountFactory } from
+    "@aa/0.6/contracts/samples/SimpleAccountFactory.sol";
+import { VerifyingPaymaster } from
+    "@aa/0.6/contracts/samples/VerifyingPaymaster.sol";
+
+contract EntryPoint0p6 is EntryPoint { }
```

Comment:
When deploying we can specify which version of entry point to deploy, I think we don't need to do this.

Reply:
This adjustment is used only in [setUp()](https://github.com/pilagod/waallet-contract/pull/29/files#diff-bc65220baf3b6c5333fafaac8b448bfbbd0b8f3e44ba232a268885ceaaf856afR36) in the foundry test because `vm.getCode()` cannot recognize different contract versions with the same name.

Or it triggers the `[FAIL. Reason: setup failed: Multiple matching artifacts found]` error.

<img width="802" alt="截圖 2024-06-05 上午11 21 17" src="https://github.com/pilagod/waallet-contract/assets/7114046/85bc77d6-b4d3-4a3e-b09f-02513703427b">

Reply:
Couldn't it get code by specific path?

Reply:
Another work around is to deploy EntryPoint first to a random address, and get code from it to etch to our favorable address.